src/Card.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './card.css';
+
+function Card (props){ 
+    const [ email, setEmail ] = React.useState('Loading...');
+    const [ firstName, setFirstName ] = React.useState('Loading...');
+    const [ lastName, setLastName ] = React.useState('Loading...');
+    const [ avatar, setAvatar ] = React.useState('Loading...');
+    
+    React.useEffect(() => {
+        async function getData() {
+            const res = await fetch(`https://reqres.in/api/users/${props.userId}`);
+            const data = await res.json();
+            const obj = await data.data;
+            
+            setEmail(obj.email);
+            setFirstName(obj.first_name);
+            setLastName(obj.last_name);
+            setAvatar(obj.avatar);
+        }
+        getData();
+    }, [props.userId]);
```

Comment:
👍 Nice work including this prop here - it's an easy thing to forget, but that can cause weird bugs if the component goes to re-render.